app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/disposaldetails/how_much_did_you_own.scala.html

```diff
@@ -91,9 +97,10 @@
         fieldId = key,
         label = pageHeading(title),
         selected = selectedIndex,
-          error = form.error(key),
+        error = form.error(key),
         hasErrors = hasErrors,
-        options = options
+        options = options,
+        customErrorKey = form.error(key).map(e => s"$userErrorPrefix${e.message}")
```

Comment:
this customErrorKey will not work - it should include the key and then the userKey like `customErrorKey = Some(s"$key$userKey")`